backend/helper_zero/views/auth.py

```diff
@@ -1,13 +1,20 @@
 from rest_framework import status, viewsets
 from rest_framework.response import Response
+from google.oauth2 import id_token
+from google.auth.transport import requests
 
 from helper_zero.models import Organization, DonationRequest
 from helper_zero.serializers import OrganizationSerializer, DonationRequestSerializer
 
 class AuthView(viewsets.ViewSet):
+
   def list(self, request):
     auth_token = request.query_params.get('auth_token')
-    org = Organization.objects.filter(auth_token=auth_token).first()
+    id_info = id_token.verify_oauth2_token(auth_token, requests.Request(), "650902157032-v1gqmd903sedgmdrpd0goa1343b049ug.apps.googleusercontent.com")
```

Comment:
is this just a custom generated string

Reply:
yeah i'll make it an env var but seems ok to expose it